src/test/java/com/synopsys/integration/alert/component/audit/web/AuditEntryActionsTest.java

```diff
@@ -152,20 +157,26 @@ public void testPagedRequest() throws AlertDatabaseConstraintException {
 
         NotificationContentRepository notificationRepository = Mockito.mock(NotificationContentRepository.class);
         AuditNotificationRepository auditNotificationRepository = Mockito.mock(AuditNotificationRepository.class);
-        JobAccessor jobAccessor = Mockito.mock(JobAccessor.class);
+        JobAccessor oldJobAccessor = Mockito.mock(JobAccessor.class);
+        JobAccessorV2 jobAccessor = Mockito.mock(JobAccessorV2.class);
+        Mockito.when(jobAccessor.getJobById(Mockito.any())).thenReturn(null);
```

Comment:
It looks like you are missing the `Mockito.when` for the `oldJobAccessor`.

Reply:
I see you do this further down. So is line 162 still necessary?
```
Mockito.doReturn(Optional.of(configuration)).when(oldJobAccessor).getJobById(Mockito.any());
        Mockito.doReturn(Optional.of(distributionJob)).when(jobAccessor).getJobById(Mockito.any());
```

Reply:
I updated `AuditEntryActions` because it used `NotificationProcessor`, but I did not need to update `DefaultAuditAccessor` for this PR.